### PR TITLE
New panel plugin to create "CDF" graphs

### PR DESCRIPTION
**Pull request by sebastiangunreben**
Hello Grafana cummunity,

I created a panel plugin to visualize the cumulative distribution function of a certain value.
I would be happy to receive your review.

BR,
Sebastian

### CODE REVIEW

**Comment by marcusolsson** [comment]
Awesome! The URL in the pull request needs to reference a zip archive containing your plugin. Could you also run that through the [plugin validator](http://plugin-validator.grafana.net/) to make sure it's ready for review?

**Comment by sebastiangunreben** [comment]
@marcusolsson … the plugin has already been validated throughout the plugin validator. Do you miss anything before the review process could start?

**Comment by marcusolsson** [comment]
I've tested the panel and successfully got it working. The only issue I found was that when I clicked "Show thresholds" with the following error:

Screenshot 2021-07-06 at 18 33 37@2x

I'm running Grafana 8.0.4 on macOS 11.2.3 with Firefox 89.

I've also configured a Community signature level for `sebastiangunreben-cdf-panel` so you should be able to sign the plugin and update your pull request.

**Comment by marcusolsson** [comment]
I'm getting an error that the plugin has a Modified signature due to `img/logo.svg` which is not included in the manifest. Any file added to the plugin after it's been signed will cause an error.

Also, could you update the pull request to only include the version you intend to publish. The currently listed versions are not valid plugin packages and won't be published, so you can go ahead and remove them from the list. Thank you! 🙏

File: repo.json [hunk 1]
```diff
@@ -4472,6 +4472,7 @@
           "download": {
             "any": {
               "url": "https://github.com/doitintl/bigquery-grafana/releases/download/2.0.2/doitintl-bigquery-datasource-2.0.2.zip",
+
```
Comment (marcusolsson) [inline repo.json:4475]: Could you remove this line?

File: repo.json [hunk 2]
```diff
@@ -6727,6 +6728,23 @@
         }
       ]
     },
+ {
+      "id": "sebastiangunreben-cdf-plugin",
```
Comment (marcusolsson) [inline repo.json:6732]:
```suggestion
      "id": "sebastiangunreben-cdf-panel",
```

File: repo.json [hunk 3]
```diff
@@ -6727,6 +6728,23 @@
...
+      "type": "app",
```
Comment (marcusolsson) [inline repo.json:6733]:
```suggestion
      "type": "panel",
```

**Review by marcusolsson — CHANGES_REQUESTED** [review]
I've done a final review of the plugin and it all looks good! Just a few fixes in the PR and I'll publish this right away!

**Comment by sebastiangunreben** [comment]
I updated pull request (instead of resolving requests via web). Hope that led to the same effect.

**Comment by marcusolsson** [comment]
Wonderful! I've published the plugin, but please allow a few minutes before it becomes available on grafana.com. 

Thanks for contributing your plugin to the Grafana community! 💪 🙏

**Comment by sebastiangunreben** [comment]
Great! Thank you very much!

**Pull request merged**